plugins/inputs/leofs/leofs.go

```diff
@@ -86,13 +86,26 @@ var KeyMapping = map[ServerType][]string{
 		"total_objects",
 		"total_size_of_active_objects",
 		"total_size",
-		"num_of_replication_messages",
-		"num_of_sync-vnode_messages",
-		"num_of_rebalance_messages",
+		"mq_num_of_msg_replication",
+		"mq_num_of_msg_sync_vnode",
+		"mq_num_of_msg_rebalance",
```

Comment:
We need to stay with the original names, or we could report it under both names if you think this is an important rename.